app/controllers/servers_controller.rb

```diff
@@ -61,6 +61,28 @@ def start_installation
     redirect_to server_path(@server)
   end
 
+  def start_update
+    @server = Server.find(params[:id])
+    if VersionInfo.parse(@server.dokku_version) < VersionInfo.parse("v0.4.11")
```

Comment:
Shouldn't we move this hard-coded reference to `v.0.4.11` to something more abstract/dynamic?